examples/complex/xss_scanner.py

```diff
@@ -202,20 +201,21 @@ def get_SQLi_data(new_body: str, original_body: str, request_URL: str, injection
         "Sybase": (r"(?i)Warning.*sybase.*", r"Sybase message", r"Sybase.*Server message.*"),
     }
     for dbms, regexes in DBMS_ERRORS.items():
-        for regex in regexes:
+        for regex in regexes:  # type: ignore
```

Comment:
Why is that required?

Reply:
Because mypy was throwing an error. More about this can be read in https://github.com/python/mypy/issues/2598 . I thought this would be the best solution as of now